ui.pyw

```diff
@@ -543,6 +550,7 @@ class EditRestaurantDialogue(RestaurantDialogue, object):
 
     def fill_initial_values(self):
         self.restaurant_text.insert(END, self.restaurant_name)
+        self.comment_text.insert(END, self.restaurant_comment)
```

Comment:
'tis better to push small commits that set a stable database before embarking on a journey of UI changes.
If I had written the new UI code and then decide that this type of comments is superfluous it means more changes. This way things are a bit more self contained.